### PR TITLE
Add GUI entry for ip_change_kill_states config parameter

### PR DESCRIPTION
**Pull request by fernsehkind**
The hidden parameter is present in pfsense since November 2014 but there was no GUI entry to enable/disable this parameter. This parameter is extremly helpful for unstable WAN connections using VOIP.

This is my first contribution to pfsense. I hope my commits are compliant to the pfsense coding style.

### CODE REVIEW

File: src/etc/rc.newwanip [hunk 1]
```diff
@@ -200,12 +200,13 @@ if (platform_booting()) {
 if (!is_ipaddr($oldip) || $curwanip != $oldip || !is_ipaddrv4($config['interfaces'][$interface]['ipaddr'])) {
 	/* IP changed, kill states accordingly */
 	if ($curwanip != $oldip) {
-		log_error("IP has changed, killing states on former IP $oldip.");
-		pfSense_kill_states($oldip);
 		if (isset($config['system']['ip_change_kill_states'])) {
-			/* hidden config option to wipe all states if needed */
-			log_error("Killing all states post-IP change.");
+			log_error("IP has changed, killing all states (ip_change_kill_states is set).");
```
Comment (jim-p) [inline src/etc/rc.newwanip]: "IP" should be "IP Address"

File: src/etc/rc.newwanip [hunk 2]
```diff
@@ -200,12 +200,13 @@ if (platform_booting()) {
...
+			pfSense_kill_states($oldip);
 			filter_flush_state_table();
+		} else {
+			log_error("IP has changed, killing states on former IP $oldip.");
```
Comment (jim-p) [inline src/etc/rc.newwanip]: "IP" should be "IP Address" in both instances

File: src/usr/local/www/system_advanced_network.php [hunk 3]
```diff
@@ -274,6 +281,15 @@
 ))->setHelp('This option will suppress ARP log messages when multiple interfaces '.
 	'reside on the same broadcast domain.');
 
+$section->addInput(new Form_Checkbox(
+	'ip_change_kill_states',
+	'Reset all states',
```
Comment (jim-p) [inline src/usr/local/www/system_advanced_network.php]: This should be title case ("Reset All States")

File: src/usr/local/www/system_advanced_network.php [hunk 4]
```diff
@@ -274,6 +281,15 @@
...
+	'Reset all states if WAN IP changes',
```
Comment (jim-p) [inline src/usr/local/www/system_advanced_network.php]: "IP" should be "IP Address"

File: src/usr/local/www/system_advanced_network.php [hunk 5]
```diff
@@ -274,6 +281,15 @@
...
+	isset($pconfig['ip_change_kill_states'])
+))->setHelp('This option will reset all states if a WAN IP changes instead of only '.
```
Comment (jim-p) [inline src/usr/local/www/system_advanced_network.php]: "will reset" should be "resets"
"if a WAN" should be "when a WAN"
"IP" should be "IP Address"

File: src/usr/local/www/system_advanced_network.php [hunk 6]
```diff
@@ -274,6 +281,15 @@
...
+    'reset states associated to old IP. This can help to kill zombie states associated '.
```
Comment (jim-p) [inline src/usr/local/www/system_advanced_network.php]: Remove "Reset" from this line.
"to old IP" should be "with the previous IP address"
Remove the last sentence (end of this line and the entire next line)

**Comment by fernsehkind** [comment]
Wording corrected.

**Pull request merged**